test/base.sh

```diff
@@ -2,7 +2,14 @@
 
 set -o pipefail
 
-export R=$(readlink -f $(dirname $0/)/../)
+PGID=$(ps -o pgid -p $$ | tail -1)
+if [ "$PGID" != "$$" ]; then
+	which setsid 1>/dev/null && exec setsid "$0" "$@"
```

Comment:
You should probably redirect stderr as well just to be complete (at least my copy of which prints errors when it doesn't find what it's looking for on Linux).  Because, of course,  `which -s` isn't universal

```
which setsid &>/dev/null && ...
```